README.md

```diff
@@ -8,15 +8,14 @@ Our default implementation works with Amazon Web Services (AWS), but AWS Amplify
   https://aws-amplify.github.io/docs/ios/start
 
 ## Features/APIs
-*Note: Amplify docs are still being updated and will go live by EOW. The below links will take you to the SDK documentation currently.
+The below links will take you to the SDK documentation currently.
```

Comment:
the links work now since docs are live so consider removing

Reply:
removed